src/components/preview/__definition__.js

```diff
@@ -0,0 +1,32 @@
+import Preview from './';
+
+let definition = new Definition('preview', Preview, {
+  description: `allows extra functionality over a basic div.`,
```

Comment:
The description needs to say what the component actually does e.g. so it makes sense when read on https://carbon.sage.com/

Reply:
👍